test/test-server/srcm_handlers.go

```diff
@@ -0,0 +1,78 @@
+package testserver
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	srcm "github.com/confluentinc/ccloud-sdk-go-v2/srcm/v2"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	id           = "lsrc-1234"
+	httpEndpoint = "https://sr-endpoint"
+	status       = "PROVISIONED"
+	packageType  = "essentials"
+	regionId     = "sgreg=1"
+)
+
+func handleSchemaRegistryClusters(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		if r.Method == http.MethodPost {
```

Comment:
Using a `switch` is preferred. It reduces boilerplate and makes it easier to add new branches.